packages/extension-vscode/src/utils/packages.ts

```diff
@@ -25,12 +28,13 @@ export const installPackages = async (packages: string[], options?: InstallOptio
 
 /**
  * Load the provided packages from the specified location.
+ * Always performs a dynamic load, even if bundled with webpack.
```

Comment:
I think you are missing the `why` here 🙂

Reply:
Indeed I am - good catch!